prod.yml

```
@@ -1,19 +1,12 @@
 version: "3.1"
 
-services:
-  ml-install:
-    container_name: ml-install
-    image: ml-install
-    volumes:
-      - ./app:/var/www/
-    command: composer install --no-dev
-  
-  ml-dist:
-    container_name: ml-dist
-    image: ml-dist
-    depends_on:
-      - "ml-install"
-    volumes:
-      - ./app:/var/www/
+services:  
+  machine-learning:
+    container_name: ml-prod
+    image: ml-prod
+    build:
+      context: .
+      dockerfile: .docker/prod/Dockerfile
     ports:
-      - "127.0.0.1:3006:80"
+      - "3006:80"
```

Il faut laisser l'application écouter sur localhost et non pas sur toutes les interfaces 

```suggestion
      - "127.0.0.0.1:3006:80"
```